source/production/F0.Minesweeper.Components/Game.razor.css

```diff
@@ -0,0 +1,3 @@
+#F0-game {
```

Comment:
(this and all other css-selectors)
I believe the "CSS" way of naming is _kebab-case_, where also the leading `f` shoud be lowercase -> `f0-game`

Reply:
Maybe we should talk again in our next session about this. I read a little bit about conventions and stumbled across [BEM and its naming conventions](http://getbem.com/naming/) and I think this would actually fit pretty nicely with Blazor and the Component based structure we're following.

Reply:
We talked about this in our last session and for now we don't want to use BEM since it adds more convention than necessary. "Just" _kebab-case_ is enough for us for now.